db/migrate/20160104131622_remove_deprecated_columns.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveDeprecatedColumns < ActiveRecord::Migration
+  def change
+    remove_column :subject_workflow_counts, :set_member_subject_id
```

Comment:
only if we run the deploy after migrating, i thought we had them as two tasks in jenkins so we could deploy before migrations?